src/components/routing/routers/AppRouter.js

```diff
@@ -6,10 +6,14 @@ import Profile from "components/views/Profile"
 import UserList from "components/views/UserList"
 import PrivateRoute from "components/views/PrivateRoute"
 import EditProfile from "components/views/EditProfile";
-import HomePage from "components/views/HomePage";
-import CreateGame from "components/views/CreateGame";
-import Lobby from "components/views/Lobby";
+
+// import Drawing from "components/views/Drawing";
```

Comment:
I think Homepage, CreateGame and Lobby shouldnt be a comment

Reply:
Yes, that's right.